icons/glue_zoom_to_rect.svg

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You don't need to copy icons that are already in glue - in those cases you can just set ``icon = 'glue_zoom_to_rect'`` (i.e. the name of the icon without extension)